resources/src/app.css

```diff
@@ -83,7 +83,7 @@ textarea {
   max-width: 100%;
   border-radius: 3px;
   border: 1px solid var(--mid-grey);
-  margin-bottom: 28px;
+  margin: 16px 0;
```

Comment:
We generally want a convention of setting margins in **one direction**

Instead of adding a margin-top here, can you accomplish the same thing by changing the margin-bottom on whatever is rendered above the `<textarea>`